packages/cli/test/integration/generators/app.integration.js

```diff
@@ -39,6 +39,9 @@ describe('app-generator specific files', () => {
     );
     assert.fileContent('src/application.ts', /constructor\(/);
     assert.fileContent('src/application.ts', /this.projectRoot = __dirname/);
+    assert.fileContent('src/application.ts', /openApiSpec: {/);
+    assert.fileContent('src/application.ts', /port: process.env.PORT || 3000/);
```

Comment:
Is this still true after moving `process.env.PORT` to `index.js`?

Reply:
You are right!, but it didn't give any error, it should have returned false, what would be happening?

Reply:
do you think the `||`  is being evaluated ?

Reply:
Yeah, you regular expression might have wildcard matches.